modules/nf-core/eggnogmapper/tests/main.nf.test

```diff
@@ -47,7 +47,60 @@ nextflow_process {
                 { assert path(process.out.annotations.get(0).get(1)).readLines().contains("ENSSASP00005000002.1\tENSSASP00005000002.1\t0.0\t14179.0\tCOG0498@1|root,COG0498@2|Bacteria,1MUWQ@1224|Proteobacteria,2VHR6@28216|Betaproteobacteria,2KUMA@206389|Rhodocyclales\t1224|Proteobacteria\tE\tthreonine synthase\t-\t-\t-\t-\t-\t-\t-\t-\t-\t-\t-\t-\t-") },
                 { assert path(process.out.orthologs.get(0).get(1)).readLines().contains("ENSSASP00005000002.1\tENSSASP00005000002.1\t0.0\t14179.0\t1\t7096\t1\t7096\t100.0\t100.0\t100.0") },
                 { assert snapshot(process.out.hits).match("hits") },
-                { assert process.out.versions }
+                { assert snapshot(process.out.versions).match("versions") }
+            )
+        }
+
+    }
+
+    test("Should search for protein annotations against the eggnogmapper db -- empty-params") {
+
+        setup {
```

Comment:
This is the same setup step as the previous test case. I think it can be written once, before the tests start.

Reply:
Thank you! I have made this change.